### ISSUE TITLE
Multitudes - Not enough gold

### ISSUE THREAD

**Issue by iwanttoeatyo** [wildbook/R2Mods#2]
In multiplayer, when other players leave the game or are dead, newly acquired gold is distributed evenly to the remaining players. So playing solo means the player should get all the gold. I think the issue is that living player count is modified by the multiplier.

I just did some tests by playing solo then opening a pill container as soon as the game starts.

| Player Count Mulitplier | Actual Gold |Expected Gold|
|--------------|--------------------------|---------------|
| 1 | 8 | 8 |
| 2 | 5 | ? |
| 3 | 4 | ? |
| 4 | 4 | 15 |
| 10 | 3 | |
| 50 | 3 | |

Here's a twich clip showing a pill container giving 15 gold when playing solo in a 4 player lobby.
https://clips.twitch.tv/HomelyPreciousMomOSkomodo

**Comment by wildbook** [comment]
Hey, I just got some time over so I checked to see what caused it.

It's more or less intentional as the goal was to play as if you had others with you, but it should be fairly easy to "fix" by adding a config value if you'd rather it'd act like you were alone in a lobby with others.
The game scales boss items dropped, gold dropped (the one you noticed), and a few other things (boss HP & boss damage in the golden portal for example) based on how many players are alive.

I'll make that a config flag, thanks for the report!
